### PR TITLE
Issue 2

### PR DESCRIPTION
**Pull request by melrick8196**
Please suggest if any changes are to be made.

### CODE REVIEW

**Comment by tapaswenipathak** [comment]
@melrick8196 Can you please attach a screenshot which shows the changes you made?

**Comment by melrick8196** [comment]
issue2

**Comment by tapaswenipathak** [comment]
Looks good, can you add for [Facebook](https://www.facebook.com/opensourcehelpcommunity/) too?

**Comment by melrick8196** [comment]
I have changed the positions of the buttons to the footer of the page, so adding more buttons in the future may not be a problem. In my opinion having too many buttons on the navbar looks clumsy.

 
issue2 1

File: oshc/templates/base.html [hunk 1]
```diff
@@ -29,6 +30,10 @@
             <li><a href="mailto:opensourcehelpcommunity@gmail.com">Contact</a></li>
             <li><a href="https://opensourcehelp.herokuapp.com/">Join Us!</a></li>
           </ul>
+          <!-- <ul class="nav navbar-nav navbar-right">
+              <li><a href=" https://github.com/OpenSourceHelpCommunity"><i class="fa fa-github" style="font-size:30px;padding-right:8px;"></i></a></li>
+              <li><a href="https://twitter.com/oshelpcommunity"><i class="fa fa-twitter" style="font-size:30px;padding-right:8px;"></i></a></li>
+          </ul> -->
```
Comment (anubhakushwaha) [inline oshc/templates/base.html]: @melrick8196 Please remove unnecessary comments :)

**Review by anubhakushwaha — COMMENTED** [review]
Thanks for your valuable contribution,just do the mentioned changes and preferably squash your commits as well,incase you need any help feel free to ping us.Thanks !

**Pull request merged**